src/VAP/BGSObjectDetector/BGSObjectDetector.csproj

```diff
@@ -1,11 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
📝 Visual Studio always wants to change these files to UTF-8+BOM when the file is saved. If you don't want this change as part of this pull request, I can eliminate it. However, once made, there should not be further churn with respect to this.